src/routes/(app)/admin[[hash=admin_hash]]/phoenixd/+page.server.ts

```diff
@@ -76,6 +77,7 @@ export const actions = {
 			.parse(Object.fromEntries(await event.request.formData()));
 
 		runtimeConfig.phoenixd.password = parsed.password;
+		runtimeConfig.phoenixd.lndAddress = await phoenixdLndAddress();
```

Comment:
The address generation should probably in the nostr certify call  - if `isPhoenixdConfigured` - and errors should be handled gracefully.

No need to store in runtimeConfig at the moment IMO

Reply:
@Tirodem suggested we store it in runtime.